digitaltwins/model_repository/src/pl/models/index.ts

```diff
@@ -1,308 +1,202 @@
 /*
  * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import { ServiceClientOptions } from "@azure/ms-rest-js";
 import * as msRest from "@azure/ms-rest-js";
 
-
 /**
- * @interface
  * An interface representing SearchOptions.
  */
 export interface SearchOptions {
-  /**
-   * @member {string} [searchKeyword]
-   */
   searchKeyword?: string;
   /**
-   * @member {ModelFilterType} [modelFilterType] Possible values include:
-   * 'interface', 'capabilityModel'
+   * Possible values include: 'interface', 'capabilityModel'
    */
   modelFilterType?: ModelFilterType;
-  /**
-   * @member {string} [continuationToken]
-   */
   continuationToken?: string;
-  /**
-   * @member {number} [pageSize]
-   */
   pageSize?: number;
 }
 
 /**
- * @interface
  * An interface representing ModelInformation.
  */
 export interface ModelInformation {
-  /**
-   * @member {string} [comment]
-   */
   comment?: string;
-  /**
-   * @member {string} [description]
-   */
   description?: string;
-  /**
-   * @member {string} [displayName]
-   */
   displayName?: string;
-  /**
-   * @member {string} [urnId]
-   */
   urnId?: string;
-  /**
-   * @member {string} [modelName]
-   */
   modelName?: string;
-  /**
-   * @member {number} [version]
-   */
```

Comment:
Did you mean to remove the comments?

Reply:
This is generated code

---
In reply to: [371559130](https://github.com/Azure/azure-iot-sdk-node/pull/681#discussion_r371559130) [](ancestors = 371559130)